prescient/util/math_utils.py

```diff
@@ -13,7 +13,10 @@ def round_small_values(x, p=1e-6):
     # Rounds values that are within (-1e-6, 1e-6) to 0.
     try:
         if math.fabs(x) < p:
-            return 0
+            return 0.0
         return x
     except:
         raise RuntimeError("Utility function round_small_values failed on input=%s, p=%f" % (str(x), p))
+
+def within_tolerance(v1, v2, tol=1e-5):
+    return math.fabs(v1 - v2) <= tol
```

Comment:
I might suggest replacing the implementation of this function with `math.isclose(v1,v2,rel_tol=0, abs_tol=tol)`, which of course is equivalent.

Reply:
I wasn't aware of that function.  I removed our `within_tolerance` and replaced it with `math.isclose`.